docs/source/get-started/package-managers.rst

```diff
@@ -3,20 +3,32 @@ Package Managers
 
 Use your favorite package manager to install Kokkos.
 
-System package managers
+Package managers
```

Comment:
I would keep generic distribution and HPC packages separate.

Reply:
Spack & conda aren't really system package mangers, I think.